ldi-nifi/ldi-nifi-processors/rdf4j-repository-put-processor/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/processors/config/RDF4JRepositoryPutMaterialisationProcessorProperties.java

```diff
@@ -0,0 +1,44 @@
+package be.vlaanderen.informatievlaanderen.ldes.ldi.processors.config;
+
+import org.apache.nifi.components.PropertyDescriptor;
+import org.apache.nifi.processor.util.StandardValidators;
+
+public final class RDF4JRepositoryPutMaterialisationProcessorProperties {
```

Comment:
PutMaterialisation sounds a bit like a hat on a hat to me.. I'd be a fan of just RDF4JRepositoryPutProcessor (so both here with "Properties" and in the processor name. Materialisation is quite a heavy term to throw around, I find)

Reply:
Will follow naming convention of original processor